### ISSUE TITLE
pkgIndex.tcl of lib/Markdown is broken

### ISSUE THREAD

**Issue by k1s2** [tobijk/caius#19]
There is an escape issue in "pkgIndex.tcl" file in lib/Markdown. Loading the package fails when the path to the library contains more than a single whitespace in series - e.g. C:\test__test\lib\Markdown". For me the following works:
```tcl
package ifneeded Markdown 1.0 "
    source \[file join [list $dir] markdown.tcl\]
    source \[file join [list $dir] version.tcl\]
"
```

**Comment by tobijk** [comment]
Hi,

thank you for reporting this. Indeed, it seems it should be as above.

Tobias

**Comment by tobijk** [comment]
Fixed in 0.16.1.
